docs/api/field_types_reference/sesselection.md

```diff
@@ -0,0 +1,23 @@
+# SesSelection
+
+This Field Type stores a single selection choice. 
+
+Unlike the [Selection Field Type](selectionfield.md),
```

Comment:
Which piece of information is more important here? I'd go with "The SesSelection Field Type is configured in a YAML file, unlike the [Selection Field Type](selectionfield.md)."
This way we shift emphasis.